pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -34,6 +36,14 @@ var (
 	invalidTolerationKeyErrFmt   = "invalid toleration key %+v: %s"
 	invalidTolerationValueErrFmt = "invalid toleration value %+v: %s"
 	uniqueTolerationErrFmt       = "toleration %+v already exists, tolerations must be unique"
+
+	// resourcePropertyNamePrefix is the prefix (also known as the subdomain) of the label name
+	// associated with all resource properties.
+	resourcePropertyNamePrefix = "resources.kubernetes-fleet.io/"
+
+	// Below is the map of supported capacity types.
+	supportedResourceCapacityTypesMap = map[string]bool{"total": true, "allocatable": true, "available": true}
```

Comment:
why not resue the const in pkg/scheduler/framework/plugins/clusteraffinity/types.go?